packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -122,27 +125,43 @@ describe('ReactHooks', () => {
       });
       return null;
     }
-    expect(() => ReactTestRenderer.create(<App />)).toThrow();
+
+    function Simple() {
+      useState(0);
+      return 123;
+    }
+    let root = ReactTestRenderer.create(null);
+    expect(() => root.update(<App />)).toThrow(OutsideHookScopeError);
+    // we want to assure that no hook machinery has broken,
+    // so we listen for all errors on the next line
+    expect(() => root.update(<Simple />)).not.toThrow();
```

Comment:
If you want to ensure everything still works, worth asserting the output is legit.